src/ContentCreator.php

```diff
@@ -615,6 +669,34 @@ protected function getEntityBundleInstances($entity_type) {
     return $this->cacheBundleInstances[$entity_type];
   }
 
+  /**
+   * Get list of bundle types with number of instances for field.
+   *
+   * TODO: Add support optional fields where 0 entities are referenced.
+   *
+   * @param string $entity_type
+   *   The entity type.
+   * @param string $bundle_type
+   *   The bundle type.
+   * @param string $field_name
+   *   The field name.
+   *
+   * @return array
+   *   The list of bundle types with number of instances.
+   */
+  protected function getEntityBundleInstancesForField($entity_type, $bundle_type, $field_name) {
+    if (isset($this->cacheBundleInstancesForField[$entity_type][$bundle_type][$field_name])) {
+      return $this->cacheBundleInstancesForField[$entity_type][$bundle_type][$field_name];
+    }
+
+    $field_target_bundle_info = $this->config[$entity_type]['_bundles'][$bundle_type]['_fields'][$field_name]['_bundle_info'];
+    $this->cacheBundleInstancesForField[$entity_type][$bundle_type][$field_name] = array_filter($this->getEntityBundleInstances($field_target_bundle_info['target_type']), function ($target_entity_instances, $target_bundle_type) use ($field_target_bundle_info) {
+      return empty($field_target_bundle_info['target_bundles']) || isset($field_target_bundle_info['target_bundles'][$target_bundle_type]);
+    }, ARRAY_FILTER_USE_BOTH);
```

Comment:
`$target_entity_instances` is not used, why use ARRAY_FILTER_USE_BOTH then?

Reply:
👍